Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml.cs

```diff
@@ -1,7 +1,16 @@
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
 namespace CO.CDP.OrganisationApp.Pages.Registration;
 
 public class OrganisationAlreadyRegistered: PageModel
 {
+    [BindProperty]
```

Comment:
Bit confused how this page is being used? `[BindProperty]` is for data that is coming from get or post data. The identifier is already sufficiently retrived from the URL segment (`@page "/registration/organisation-already-registered/{Identifier}"`) by the `OnGet(string identifier)` so you don't need the Bind Property at all?